src/shared/components/MultiUserSelectDropdown/MultiUserSelectDropdown.tsx

```diff
@@ -62,12 +56,12 @@ export const MultiUserSelectDropdown: FunctionComponent<MultiUserSelectDropdownP
 	useEffect(() => {
 		if (values.length) {
 			UserService.getNamesByProfileIds(values)
-				.then((users: Avo.User.User[]) => {
+				.then((users: Partial<CommonUser>[]) => {
 					setSelectedProfiles(
 						users.map(
 							(user): PickerItem => ({
 								label: `${get(user, 'full_name')} (${get(user, 'mail')})`,
-								value: user.profile?.id || '',
+								value: user.profileId || '',
```

Comment:
Maybe try use the same way of working throughout the project/file. Line 63 uses lodash to get a. value so maybe change that one to `${user.full_name} ${user.mail}` if you want to get rid of lodash.